test/README.md

```diff
@@ -1,47 +1,59 @@
 $ make clean; make
-$ ./run_unit_bmi_test ./data/topmod_100.run
-```
-The configuration file used here considers 100 timesteps of size 1 hour, but the actual testing loop is much smaller. 
-To fully test the depth of all BMI functions, we are using a `stand_alone` value of `FALSE`.  See note on [`stand_alone`](#stand-alone-flag).
+$ ./run_bmi_unit_test
+``` 
+Note: Older compilers may require you to first run `module load gnu` prior to running `make`.
+
+To fully test the depth of all BMI functions, we are using a `stand_alone` value of `FALSE` (`0`).  See note on [`stand_alone`](#stand-alone-flag).
 Recall that BMI guides interoperability for model-coupling, where model components (i.e. inputs and outputs) are easily shared amongst each other.
 When testing outside of a true framework, we consider the behavior of BMI function definitions, rather than any expected values they produce. 
 
-# Comparison
+## Comparison
 To compare bmi-enabled model results to output produced by original source code, we can run TOPMODEL in a `stand_alone` scenario. 
-BMI model control functions (`initialize()`, `update()`, and `finalize()`) are specialized to manage this conditional switch. Other BMI functions primarily support use 
-within a model-coupling framework.  Therefore, these “self-description” functions are unneeded for expected value tests.
+BMI model control functions (`bmi.initialize()`, `bmi.update()`, and `bmi.finalize()`) are specialized to manage this conditional switch. 
+Other BMI functions primarily support use within a model-coupling framework.
+Therefore, these “self-description” functions are unneeded for expected value tests.
 
+From the project's root directory,
 ```
-$ cd alt-modular/Modules/TOPMODEL/src/
+$ cd src/
 $ make clean; make
+$ cd ..
 $ ./run_bmi
 ```
-The executable is configured for a `stand_alone` scenario and reads-in information the same as the previous source code for TOPMODEL.  The two output files, `topmod.out` and `hyd.out`
-are identical to those produced by the original source code, prior to any BMI implementation.  The results are a 1-1 match, as shown in the plot below.  Instructions on how to run original TOPMODEL can be
-found [here](https://github.com/NOAA-OWP/alt-modular/blob/main/Modules/TOPMODEL/references/1_ORG_CODE_C_FO/README.1ST).
+The executable is configured for a `stand_alone` scenario and reads-in information the same as the previous source code for TOPMODEL.
+The two output files, `topmod.out` and `hyd.out` are identical to those produced by the original source code, prior to any BMI implementation.
+The results are a 1-1 match, as shown in the plots below.
+Instructions on how to run original TOPMODEL can be found [here](../refs/original_code_c/README.1ST).
+See [OUTPUT_FILES_EXPLAINED](../docs/OUTPUT_FILES_EXPLAINED.md) or [OUTPUT_FILES_EXPLAINED](../docs/OUTPUT_FILES_EXPLAINED.md) [source code](../refs/original_code_c/tmod9502.c) for more information about output variable names and units.
```

Comment:
repeated OUTPUT_FILES_EXPLAINED.  delete second one (before "source code")

Reply:
I meant to have `INPUT_FILES_EXPLAINED` will adjust.

Reply:
actually, this should just be removed, you're correct.